pkg/services/ngalert/database_mig.go

```diff
@@ -107,3 +107,19 @@ func alertInstanceMigration(mg *migrator.Migrator) {
 	mg.AddMigration("add index in alert_instance table on def_org_id, def_uid and current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[0]))
 	mg.AddMigration("add index in alert_instance table on def_org_id, current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[1]))
 }
+
+func alertmanagerConfigurationMigration(mg *migrator.Migrator) {
+	alertConfiguration := migrator.Table{
+		Name: "alert_configuration",
+		Columns: []*migrator.Column{
```

Comment:
Configurations will be shared among all the organisations? If not, then there should be an additional column `org_id`.

Reply:
That's my current assumption, only 1 configuration for all organisations.